src/olympia/amo/urlresolvers.py

```diff
@@ -238,12 +240,9 @@ def linkify(match):
     return URL_RE.sub(linkify, unicode(jinja2.escape(text)))
 
 
-def linkify_with_outgoing(text, nofollow=True, only_full=False):
+def linkify_with_outgoing(text):
```

Comment:
This strictly isn't necessary for the Bleach update, but I simplified this function down to what was actually used in the code.